src/main/java/pl/edu/icm/pl/mxrdr/extension/importer/xds/XdsAnalysisField.java

```diff
@@ -0,0 +1,49 @@
+package pl.edu.icm.pl.mxrdr.extension.importer.xds;
+
+
+public enum XdsAnalysisField {
```

Comment:
Yes, the idea is that it could also be informative for the user that something was not extracted (otherwise s/he will not even notice that). It may also be a feedback for us (i.e. that there is something wrong with the parser and we're not catching some edge cases).